includes/kco-functions.php

```diff
@@ -690,7 +690,12 @@ function kco_validate_order_total( $klarna_order, $order ) {
  */
 function kco_validate_order_content( $klarna_order, $order ) {
 	$order_data = new KCO_Request_Order();
+	$prefix     = "Klarna order ID: {$klarna_order['order_id']} | WC Order ID: {$order->get_order_number()}:";
 
+	// An array of notes to display to the merchant.
+	$notes = array( __( 'A mismatch between the WooCommerce and Klarna orders was identified. Please verify the order in the Klarna merchant portal before processing.', 'klarna-checkout-for-woocommerce' ) );
+
+	// A match happens when the item reference and quantity matches in Woo and Klarna.
 	$mismatch = false;
 	foreach ( $order->get_items() as $order_item ) {
```

Comment:
I don't see an logging for if a order line was not found at all. Please add that as well.